src/main/resources/templates/db_scripts/relatedInstanceType.sql

```diff
@@ -0,0 +1,53 @@
+DROP FUNCTION IF EXISTS set_related_instance_type_md_json;
+
+CREATE FUNCTION set_related_instance_type_md_json ()
```

Comment:
It's RMB's task to create and upgrade these functions. If there is a bug in RMB it should be fixed in RMB (because it affects other modules as well).

Reply:
@julianladisch please can you assist with investigation into the report that those functions are not included in a clean installation of the module.